src/css/custom.css

```diff
@@ -491,3 +524,129 @@ a.table-of-contents__link.toc-highlight > span.badge {
 .tabs-hidden {
   display: none;
 }
+
+/* Page titles */
+.markdown h1:first-child {
+  font-weight: 600;
+  font-size: 2.5rem;
+}
+h1 {
+  padding-top: 10px;
+  font-family: "IBM Plex Sans", "sans-serif";
+}
+h1.openapi__heading {
+  margin-bottom: calc(
+    var(--ifm-h1-vertical-rhythm-bottom) * var(--ifm-leading)
+  ) !important;
+}
+/* Main page headings */
+h2 {
+  font-size: 28px;
+  font-weight: 600;
+  font-family: "IBM Plex Sans", sans-serif;
+  border-bottom: 1px solid #dedede;
```

Comment:
blocking: 

This does strange things to the API explorers. (1) Worst, a line that spans only the text of the "Responses" heading; (2) Not as bad but still unpleasant, two horizontal lines surrounding the heading "Responses". 

<img width="643" alt="image" src="https://github.com/user-attachments/assets/6987b90a-c660-4815-814c-e6a5cb9efabe">

(from https://preview.docs.camunda.cloud/pr-4324/docs/next/apis-tools/camunda-api-rest/specifications/fail-job/)

I don't think I have a specific suggestion to resolve this, but I do think this runs counter to your goal of making the docs look more professional.

Reply:
Fixed (hopefully) - pretty tricky to target this and had to remove padding from above tabs, but should be good now 👍

Reply:
This might be what you're talking about in your comment re: differences between local and preview, but if not, this is what I'm seeing now, and I don't think you were going for a line under just the response code tabs: 

<img width="655" alt="image" src="https://github.com/user-attachments/assets/49b8a8af-6168-4c0b-a438-fea00c7872a5">